### PR TITLE
Add additional networks rpc endpoints

### PR DESCRIPTION
**Pull request by Odraxs**
## Description

Add support to testnet and local RPC endpoints when the requests are performed.  Allowing to work with testnet and a local network via env configurations. 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -199,6 +199,12 @@ Struct.new([field1, field2])
 
 Interaction with the Soroban-RPC server is done through the `Soroban.RPC` module.
 
+To interact with another network than the futurenet, you can change the `:stellar_sdk` config, which will also change to the RPC endpoint that works with that network. e.g.
```
Comment (einerzg) [inline README.md:202]: Currently, Soroban.ex can run on both Futurenet and Testnet, the former being the default. To use the Testnet network change the `:stellar_sdk` configuration, which will also change to the RPC endpoint that works with that network. e.g.

Comment (einerzg) [reply README.md:202]: How is it going?

File: lib/rpc/request.ex [hunk 2]
```diff
@@ -7,7 +7,11 @@ defmodule Soroban.RPC.Request do
 
   alias Soroban.RPC.{Client, Error, HTTPError}
 
-  @default_url "https://rpc-futurenet.stellar.org:443/"
+  @base_urls [
+    test: "https://soroban-testnet.stellar.org",
+    future: "https://rpc-futurenet.stellar.org",
+    local: "http://localhost:8000"
```
Comment (einerzg) [inline lib/rpc/request.ex:13]: do we really need the local network?

**Review by einerzg — CHANGES_REQUESTED** [review]

File: lib/rpc/request.ex (hunk 2 above)
Comment (Odraxs) [reply lib/rpc/request.ex:13]: @einerzg Currently `stellar_sdk` already supports local so we didn't find any reason to not include it here.

**Review by einerzg — APPROVED** [review]

**Pull request merged**